src/components/app/Home.css

```diff
@@ -206,7 +206,7 @@
 
   .homeAdditionalContent,
   .homeInstructions {
-    grid: none;
+    grid-template-columns: minmax(0, 1fr);
```

Comment:
Could you also add the comment here as well that explains minmax? I don't think it's very clear without a comment.